CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## 1.12.0 (February 28, 2024)
+ * [#3880](https://github.com/redhat-developer/vscode-openshift-tools/pull/3880) Do not call getAllComponents() in activate.
+ * [#3112](https://github.com/redhat-developer/vscode-openshift-tools/issues/3112) Extension takes several minutes to start if the current `~/.kube/config` context is an inaccessible cluster.
+ * [#3931](https://github.com/redhat-developer/vscode-openshift-tools/issues/3931) New Serverless Function is not shown after creation.
+ * [#3928](https://github.com/redhat-developer/vscode-openshift-tools/issues/3928) Can't load Helm chart.
+ * [#3879](https://github.com/redhat-developer/vscode-openshift-tools/pull/3879) When turned to using service account for a Sandbox a wrong warning me….
```

Comment:
```suggestion
 * [#3879](https://github.com/redhat-developer/vscode-openshift-tools/pull/3879) When turned to use a service account for a Sandbox a wrong warning message appears.
```

Reply:
Done
Plus  one  more commit title is extracted:

```
- * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project....
+ * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project when new project name is the same as the one set in the current-context.
```